app/controllers/api/base_controller.rb

```diff
@@ -54,5 +53,50 @@ def set_access_control_headers
     def collection_config
       @collection_config ||= CollectionConfig.new
     end
+
+    def log_request_initiated
```

Comment:
My feeling was that I prefer to see this closer to where it is called [here](https://github.com/ManageIQ/manageiq/pull/15521/files/342ae5e783eea7e3af9028d34a77b5400c8e4e78#diff-b4f92b4e361ffab4fbb91c68eb3f77bbL29), but can keep it in a concern if that is preferable.